shared/reducers/config.js

```diff
@@ -74,7 +76,8 @@ export default function (state: ConfigState = initialState, action: Action): Con
         return {
           ...state,
           status,
-          username: status.user && status.user.username
+          username: status.user && status.user.username,
+          loggedIn: status.loggedIn
```

Comment:
also we hoist loggedIn into the root. I think we actually want this for all props in status, as having status/config/extendedConfig as raw values from the rpc isn't what we really want in our store anyways